### PR TITLE
Trim galore 0.4.3

### PR DESCRIPTION
**Pull request by hsiaut**
* [x ] I have read the guidelines above.
* [ x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Added version 0.4.3 to trim_galore
This is my first recipe, thanks.

@bioconda/core

### CODE REVIEW

**Comment by bgruening** [comment]
Welcome @hsiaut! We usually put the latest version in the root trim_galore folder and older versions into sub directories.

File: recipes/trim_galore/0.4.3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  number: 0
+  skip: True # [osx]
+source:
+  fn: trim_galore_v0.4.3.zip
+  url: http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/trim_galore_v0.4.3.zip
+  md5: 84dec799ae1a2d9a11d57de06a9fdfb8
+requirements:
+  build:
+  run:
+    - perl-threaded
```
Comment (bgruening) [inline recipes/trim_galore/0.4.3/meta.yaml]: We switched recently to use `perl`

**Comment by hsiaut** [comment]
Thanks, I changed those items you pointed out.

**Comment by bgruening** [comment]
Awesome! Look at the diff, so much easier to review now :)
Let's wait for travis and see if everything is ok!

Thanks again!
Bjoern

**Comment by hsiaut** [comment]
Thanks, the Travis CI failed and I'm not sure why.

On my Travis CI it worked.

Based on the log, it was able to download the source file okay, unpack, and set permissions.  Not sure what comes in the testing after that. 

https://travis-ci.org/bioconda/bioconda-recipes/jobs/205926268

**Comment by bgruening** [comment]
The error is here: https://travis-ci.org/bioconda/bioconda-recipes/jobs/205926268#L525

There are some script that have `/usr/bin/perl` hardcoded. Can you patch this to `/usr/bin/env perl`?

**Comment by hsiaut** [comment]
Thanks!  I just did that.

**Comment by bgruening** [comment]
And it turns green! :)
Thanks!

**Pull request merged**